qiskit_experiments/framework/experiment_data.py

```diff
@@ -1639,6 +1639,10 @@ def _wait_for_futures(
                     name,
                     self.experiment_id,
                 )
+                value = False
+            elif not fut.result()[1]:
+                LOG.debug("An error occurred for %s [Experiment ID: %s]", name, self.experiment_id)
```

Comment:
I don't think this error message is necessary. When analysis callback fails, you have dedicated message here
https://github.com/Qiskit/qiskit-experiments/blob/39cac2d32a31fe828fbdcdc7f226912d16ba678a/qiskit_experiments/framework/experiment_data.py#L914-L920
and when job fails, you have another message here
https://github.com/Qiskit/qiskit-experiments/blob/39cac2d32a31fe828fbdcdc7f226912d16ba678a/qiskit_experiments/framework/experiment_data.py#L807-L816
so this message just messes up the error log. Instead you can write a comment why this is check is added with a link to the issue so that future maintainer doesn't remove this block.

Reply:
Done in https://github.com/Qiskit/qiskit-experiments/pull/871/commits/a67d097885ab29725a4bb140348ccbaf3e9a43cc